datahub-web-react/src/app/entity/shared/containers/profile/__tests__/EntityProfile.test.tsx

```diff
@@ -105,7 +105,7 @@ describe('EntityProfile', () => {
         await waitFor(() =>
             expect(getByText('This and here we have yet another Dataset (YAN). Are there more?')).toBeInTheDocument(),
         );
-    });
+    }, 10000);
```

Comment:
why is this 10 second timeout extension necessary here? does this schema field change affect our ability to render this?

Reply:
I think it's just my local machine is too slow. I checked in to help people who also has a slow machine. Let me revert the timeout extension change since it concerns you.

Reply:
thank you! yeah it's not a big deal, I just wanted to make sure we weren't introducing a change that required this here (which I would not have guessed that to be the case by the changes you've made)